specification/maps/data-plane/Creator/preview/2022-09-01-preview/features.json

```diff
@@ -0,0 +1,959 @@
+        "operationId": "Features_GetItems",
+        "x-ms-examples": {
+          "GetFeatures": {
+            "$ref": "./examples/features/Features_GetItems.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "./dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/LimitFeatures"
+          },
+          {
+            "$ref": "#/parameters/Bbox"
+          },
+          {
+            "$ref": "#/parameters/Filter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ExtendedGeoJsonFeatureCollection"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "post": {
+        "description": "The Post Feature API creates a feature with the provided attributes in the given collection.\n\nThe Features API is a part of [Creator](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps), which makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v20220901preview/dataset/create). Datasets consist of multiple feature collections, which are a collection of features of a similar type based on a common schema referred to as ontology. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).",
```

Comment:
Is it necessary to define the different geometry types, after we have linked to the Geojson Doc. I.e., can we remove this: 
> GeoJsonGeometry:
GeoJsonPoint
GeoJsonMultiPoint
GeoJsonLineString
GeoJsonMultiLineString
GeoJsonPolygon
GeoJsonMultiPolygon
GeoJsonGeometryCollection

because we have a link to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.1) in the description (under Request Body and under Feature Definition)

Reply:
These definitions are coming from [common.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/maps/data-plane/Common/preview/1.0/common.json#L184). Although it may be overly precise, since it's a commonly referenced file, I think it's best to keep it consistent across our different swagger docs.